includes/utils/class-amp-html-utils.php

```diff
@@ -52,11 +52,12 @@ public static function is_valid_json( $data ) {
 			$decoded = json_decode( $data );
 			if ( function_exists( 'json_last_error' ) ) {
 				return ( json_last_error() === JSON_ERROR_NONE );
-			} else {
-				// For PHP 5.2 back-compatibility.
-				return null !== $decoded;
```

Comment:
Note that `json_last_error()` was introduced in PHP 5.3, and since PHP 5.4 is the minimum-supported version, we don't need this method anymore. It can be just:



This entire method could be just replaced with:

```
public static function is_valid_json( $data ) {
    json_decode( $data );
    return json_last_error() === JSON_ERROR_NONE;
}
```